package-managers/nuget.md

```diff
@@ -6,10 +6,65 @@ TODO (inspiration: http://docs.myget.org/docs/walkthrough/getting-started-with-n
 
 public feed, private feed with basic auth, pre-authenticated feed URL
 
-### Credential Provider for Visual Studio
+### Find our feed URL
+
+#### Private feed
+
+Private feeds are only accessible to the feed owner, and to other users who were granted access by the feed owner. When we need to connect to a private feed we don't own, we may need to ask the feed owner to invite us to the feed.
+
+<p class="alert alert-success">
+    <strong>Tip:</strong> We can find out how to invite users <a href="../inviting-users-to-a-feed.md">here</a>
+</p>
+
+![Feed details](assets/feed-details.png)
+
+<p class="alert alert-success">
+    <strong>Tip:</strong> We support <a href="#visualstudio">Visual Studio</a>, <a href="#rider">JetBrains Rider</a>, the <a href="#nugetcli">NuGet CLI</a>, <a href="#dotnetcli">dotnet CLI</a> and <a href="#paket">Paket</a>

```

Comment:
`, Paket and any other NuGet clients that use the standard NuGet v2 or v3 protocols`.